ratpack-core/src/test/groovy/ratpack/core/http/client/HttpClientRedirectionSpec.groovy

```diff
@@ -188,6 +223,78 @@ class HttpClientRedirectionSpec extends BaseHttpClientSpec {
     pooled << [true, false]
   }
 
+  def "can follow a relative redirect get request with encoded path parameters"() {
```

Comment:
For this test case, I think there are two useful values to assert on: the path (as currently done) and the path token (`render context.pathTokens.val`).  The former should be preserved encoded, while the latter should IMO be decoded.  From a quick look, I think this behavior is worked after your changes.

Reply:
Added.